src/Serilog.Enrichers.Environment/Enrichers/EnvironmentUserNameEnricher.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2013-2015 Serilog Contributors
+﻿// Copyright 2013-2016 Serilog Contributors
```

Comment:
(Note for later: the conditionals in this file need to be sorted out as part of #638 - the new style is `#if ENVIRONMENT` for example.)